apps/database/supabase/migrations/20260504120000_add_user_workspace_configs.sql

```
@@ -0,0 +1,43 @@
+    value TEXT NOT NULL,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT now(),
+    updated_at TIMESTAMPTZ NOT NULL DEFAULT now(),
+    PRIMARY KEY (user_id, ws_id, id)
+);
+
+CREATE INDEX IF NOT EXISTS idx_user_workspace_configs_ws_id
+    ON public.user_workspace_configs(ws_id);
+
+ALTER TABLE public.user_workspace_configs ENABLE ROW LEVEL SECURITY;
+
+CREATE POLICY "Users can manage their own workspace configs" ON public.user_workspace_configs
```

SQLFluff (4.1.0)

[error] Do not use special characters in identifiers.

(RF05)